### DISCUSSION TITLE
PS Execution Policy

### DISCUSSION THREAD

**Discussion by XXLMandalorian013** [microsoft/winget-pkgs#87810] [Ideas]
Hello,

I was getting ready to start my journey in creating a package for WinGet following the instructions [here,](https://github.com/microsoft/winget-pkgs/blob/master/doc/tools/YamlCreate.md) and noticed the  ```YamlCreate.ps1```  is not remote signed.

``` .\YamlCreate.ps1: File G:\GoogleDrive\Repos\winget-pkgs\Tools\YamlCreate.ps1 cannot be loaded. The file G:\GoogleDrive\Repos\winget-pkgs\Tools\YamlCreate.ps1 is not digitally signed. You cannot run this script on the current system. For more information about running scripts and setting execution policy, see about_Execution_Policies at https://go.microsoft.com/fwlink/?LinkID=135170. ```

Our current Org Machine policy is Local & RemoteSigned via GPO. I had to change the GPO for this machine to unrestricted temporarily to be able to run the ```YamlCreate.ps1```  . 

I propose someone on the project remote sign the .ps1.

Is there something I could be doing differently?

Was the expectation for me to remote sign it myself after cloning?

Look forward to hearing back!

**Comment by vedantmgoyal9** [comment]
The script is maintained by the community and frequent changes are made to it. If I remember correctly, the code signature changes every time, the script is modified, which is why it has not been digitally signed.

**Reply by XXLMandalorian013** [reply]
I was under the impression the .ps1 was a template and prompted for input creating a package. I have not messed with remote signing and after combing through the .ps1, I didn't see anything regarding unrestricted or a signature block. It's all a bit over my head. Going to try [WinGetCreate](https://github.com/microsoft/winget-create) and circle back if needed.
